build.sh

```diff
@@ -52,6 +52,9 @@ dockerbuild()
     BUILD_COMMAND=/opt/code/build_projects/dotnet-cli-build/build.sh $DIR/scripts/dockerrun.sh --non-interactive "$@"
 }
 
+# Initialize the tooling
+$DIR/init-tools.sh "${args[@]}"
```

Comment:
long term I think we need some kind of outer shell build.sh and an inner build.sh that gets called in docker (not so different from what we have today).

Or we merge init-tools.sh with whatever the inner build.sh is, and just call that in docker, but then it would also need to invoke `dotnet build3`